charts/horizon/values.yaml

```diff
@@ -49,7 +49,8 @@ config:
       server: http://el-horizon-listener:8080
       namespace: "{{ .Release.Namespace }}"
       kubeconfig: ""
-      s3:
+      logStorage:
+        type: "{{ if .Values.tags.full }}s3{{ else }}dummy{{ end }}"
         accessKey: admin
```

Comment:
When using dummy, fields like accessKey can be deleted.

Reply:
The configs are reserved for user to install horizon in full mode, which needs minio.